src/Val.cc

```diff
@@ -3963,28 +3963,26 @@ ValManager::ValManager()
 	for ( auto i = 0u; i < PREALLOCATED_INTS; ++i )
 		ints[i] = Val::MakeInt(PREALLOCATED_INT_LOWEST + i);
 
-	for ( auto i = 0u; i < ports.size(); ++i )
-		{
-		auto& arr = ports[i];
-		auto port_type = (TransportProto)i;
-
-		for ( auto j = 0u; j < arr.size(); ++j )
-			arr[j] = IntrusivePtr{AdoptRef{}, new PortVal(PortVal::Mask(j, port_type))};
-		}
 	}
 
-const PortValPtr& ValManager::Port(uint32_t port_num, TransportProto port_type) const
+const PortValPtr& ValManager::Port(uint32_t port_num, TransportProto port_type)
 	{
 	if ( port_num >= 65536 )
 		{
 		reporter->Warning("bad port number %d", port_num);
 		port_num = 0;
 		}
 
-	return ports[port_type][port_num];
+	std::pair key{port_num, port_type};
+	if ( ports.find(key) == ports.end() )
+		{
```

Comment:
These braces can be removed (or run clang-format, which will likely remove them).

Reply:
ran clang format.